React/Base/RCTUtilsUIOverride.h

```diff
@@ -5,6 +5,13 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+#import <Foundation/Foundation.h>
+#if TARGET_OS_OSX // TODO(macOS GH#774)
+#import <React/RCTUIKit.h>
+#else // [TODO(macOS GH#774)
+#import <UIKit/UIKit.h>
```

Comment:
We shouldn't need this. <React/RCTUIKit.h> should be pulling in UIKit.h already

Reply:
Resolved.